content.md

```diff
@@ -1555,23 +1555,25 @@ combat.
 **7. End of Round:** End of round cards and effects are played here.
 
 
-
 The acting minion always gets first opportunity to use cards or effects
 before the opposing minion at every stage of combat.
 
-
+At any point, if one or both of the combattants are no longer ready, 
+that is if their status is being changed from ready to non-ready, proceed to step 7, End of Round.
+For instance, if a minion is subject to an effect that "sends to torpor", "burns", or "removes from the game", 
+they are considered as "going to torpor", "being burned" or "being removed from the game" respectively.
+The change of status of the minion will be effective once combat is over (for instance, an ally being burned will be in the ash heap only after combat).
 
```

Comment:
```suggestion
they are considered as "going to torpor", "being burned" or "being removed from the game" respectively.\
If a vampire cannot mend their wounds, is sent to torpor or is otherwise wounded, they are no longer ready and go into torpor (if this happens during combat, they are considered as "going to torpor" until the combat is over).
```